src/win32/sys_time.h

```diff
@@ -12,15 +12,8 @@ struct timezone {
     int tz_dsttime;
 };
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 int gettimeofday(struct timeval *tv, struct timezone *tz);
 
-#ifdef __cplusplus
-}
-#endif
 
```

Comment:
```suggestion
```

Reply:
Addressed in commit da3d9e3f1e1a6d116df5830cf506f30ff56196b3.